File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.py
import sys
class Solution:
    def minSubArrayLen(self, target: int, nums: List[int]) -> int:
        minlength = sys.maxsize
        windowSum = 0
        idx = 0
        left = 0
        while idx < len(nums):
            windowSum += nums[idx]
            print(windowSum)
            
            while windowSum >= target:
                minlength = min(minlength, idx - left + 1)
                windowSum -= nums[left]
                left += 1

            idx += 1
            
        return 0 if minlength == sys.maxsize else minlength
            

File: Data Structures & Algorithms/two-integer-sum/submission-1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for(int i=0;i<nums.Length;i++){
            for(int j=i+1;j<nums.Length;j++){
                int expect = nums[i]+nums[j];
                if(expect == target){
                    return new int[]{i,j};
                }
                else{
                    continue;
                }
            }
        }
        return new int[]{0,0};

    }
}
